deployer/config_validation.py

```diff
@@ -152,3 +152,47 @@ def validate_support_config(cluster_name):
                 sys.exit(1)
     else:
         print_colour(f"No support defined for {cluster_name}. Nothing to validate!")
+
+
+def validate_authenticator_config(cluster_name, hub_name):
+    """
+    Validates the provided authenticator config for each hub of a specific cluster.
```

Comment:
Could you add a few more comments throughout this function to in plain english what is being checked/validated in each major step?